### PR TITLE
fix: changing color of beta tag for Service Map & light theme changer

### PR DESCRIPTION
**Pull request by pranay01**
### Summary

Changes the color of tag for Service Map from `magenta` to `geekblue`

Before:
before

After:
after

### CODE REVIEW

File: frontend/src/container/SideNav/NavItem/NavItem.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function NavItem({
 
 				{isBeta && (
 					<div className="nav-item-beta">
-						<Tag color="magenta">Beta</Tag>
+						<Tag color="geekblue">Beta</Tag>
```
Comment (vikrantgupta25) [inline frontend/src/container/SideNav/NavItem/NavItem.tsx:34]: let's use our design token variables here. `Color.XYZ`

Comment (pranay01) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx:34]: How do I find the design token variables? @vikrantgupta25

Comment (vikrantgupta25) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx:34]: https://github.com/SigNoz/signoz-core-ui/blob/main/packages/tokens/src/Colors/Colors.ts

**Comment by vikrantgupta25** [comment]
image
better to update here as well to maintain uniformity ?

**Comment by pranay01** [comment]
> image
> should we update here as well to maintain uniformity ?

Yes, will update here as well. Let me first figure out how to use color from design tokens

File: frontend/src/container/SideNav/NavItem/NavItem.tsx (hunk 1 above)
Comment (pranay01) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx:34]: I tried using custom color, but AntD tag component by default gives a background for custom color

`<Tag bordered={false} color={Color.TEXT_ROBIN_500}>Beta</Tag>
`
custome

I think let's just use `geekblue`

**Review by vikrantgupta25 — APPROVED** [review]

File: frontend/src/container/SideNav/NavItem/NavItem.tsx (hunk 1 above)
Comment (vikrantgupta25) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx]: just checked they support only their preset colors in the `color` prop hence won't work , approved ✅

File: frontend/src/container/SideNav/NavItem/NavItem.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import './NavItem.styles.scss';
 import { Tag } from 'antd';
 import cx from 'classnames';
 
+// import { Color } from '@signozhq/design-tokens';
```
Comment (YounixM) [inline frontend/src/container/SideNav/NavItem/NavItem.tsx:8]: Remove commented code.

File: frontend/src/container/SideNav/NavItem/NavItem.tsx [hunk 3]
```diff
@@ -31,7 +32,21 @@ export default function NavItem({
...
 				{isBeta && (
 					<div className="nav-item-beta">
-						<Tag color="magenta">Beta</Tag>
+						{/* Ways of adding color beased on our design tokens.  Tried using custom color, but AntD tag component by default gives a background for custom color */}
+						{/* <Tag bordered={false} color={Color.TEXT_ROBIN_400}>Beta</Tag> */}
+
+						<Tag bordered={false} color="geekblue">
+							Beta
+						</Tag>
+						{/* <Tag
```
Comment (YounixM) [inline frontend/src/container/SideNav/NavItem/NavItem.tsx:41]: Remove commented code.

File: frontend/src/container/SideNav/NavItem/NavItem.tsx (hunk 2 above)
Comment (pranay01) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx:8]: I wanted to keep commented code on how to used design-tokens in future. Is that not recommended?

Comment (pranay01) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx]: removed commetns

File: frontend/src/container/SideNav/NavItem/NavItem.tsx (hunk 3 above)
Comment (pranay01) [reply frontend/src/container/SideNav/NavItem/NavItem.tsx]: removed comments

**Comment by pranay01** [comment]
@YounixM addressed your comments. Can you approve and merged

**Review by YounixM — APPROVED** [review]

**Pull request merged**